docs/releases/1.10.txt

```diff
@@ -324,6 +324,9 @@ Templates
 
 * Added the ``is`` comparison operator to the :ttag:`if` tag.
 
+* The :tfilter:`urlize` template filter now handles multiple trailing
```

Comment:
I guess it doesn't hurt to mention, but personally I wouldn't consider this much of a "feature". :-)